### PR TITLE
Add telemetry event for content provider call for getting enrollment id

### PR DESCRIPTION
**Pull request by iamgusain**
### What
Add telemetry event for content provider call for getting enrollment id

### Why
Need a way to measure if company portal was called to get enrollment id and how much time it took. 

### How
Added a new TelemetryEventType for ContentProviderCall, and emitting it when we make a call to Company portal for getting enrollment id

## Testing
Verified locally that the event is getting added to the event map

### CODE REVIEW

**Review by nickbopp — APPROVED** [review]

**Comment by nickbopp** [comment]
@iamgusain happy to sign off when this is moved to a full review.

**Review by bktowett — APPROVED** [review]

File: common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ContentProviderCallEvent.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.telemetry.events;
+
+import com.microsoft.identity.common.java.telemetry.TelemetryEventStrings;
+
+import lombok.NonNull;
+
+public class ContentProviderCallEvent extends BaseEvent {
```
Comment (mohitc1) [inline common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ContentProviderCallEvent.java:32]: Consider adding documentation for the class.

**Review by mohitc1 — APPROVED** [review]
:shipit:

File: common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ContentProviderCallEvent.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public ContentProviderCallEvent(@NonNull final String contentUri){
```
Comment (mohitc1) [inline common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ContentProviderCallEvent.java:37]: Should we take a name too as parameter, as it seems this event could be used for different ContentProvider scenarios and each with different name would be helpful. e.g. this case GetEnrollmentId ?

Comment (iamgusain) [reply common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ContentProviderCallEvent.java:37]: content uri should be different for each cases, that should be able to differentiate.

**Pull request merged**